EMS/Program.cs

```diff
@@ -0,0 +1,304 @@
+﻿using System;
+using CommandLine;
+using CommandLine.Text;
+using Microsoft.Extensions.Configuration;
+using System.Text.Json;
+using System.Text.RegularExpressions;
+
+namespace EmployeeManagement;
+
+public class Options
+{
+    [Option('o', "operation", Required = true, HelpText = "Operation to perform: add, display, update, delete, filter, help")]
+    public string Operation { get; set; }
+
+    [Option('i', "identifier", Required = false, HelpText = "Identifier for operation (e.g., employee ID)")]
+    public string Identifier { get; set; }
+
+    [Option('h', "help", HelpText = "Display this help screen")]
+    public bool Help { get; set; }
+}
+
+public class Program
+{
+    private IConfigurationRoot _configuration;
+    private ILogger _logger;
+    private IEmployeeDal _employeeDal;
+    private IEmployeeBal _employeeBal;
+    private readonly string _filePath = "";
+
+    public Program()
+    {
+        _configuration = new ConfigurationBuilder()
+       .SetBasePath(Directory.GetCurrentDirectory())
+       .AddJsonFile("appsettings.json", optional: false, reloadOnChange: true)
+       .Build();
+        _logger = new Logger();
+        _filePath = _configuration["AppSettings:FilePath"];
+        _employeeDal = new EmployeeDal(_logger, _filePath);
+        _employeeBal = new EmployeeBal(_logger, _employeeDal);
+    }
+
+    private Employee EmployeeInput()
+    {
+        long? mobileNumber = 0;
+        string? employeeNumber, firstName, lastName, dateOfBirth, joiningDate, emailId, location, jobTitle, department, manager, project;
```

Comment:
why do you need these variables? you can directly create employee object and use it for storage right.